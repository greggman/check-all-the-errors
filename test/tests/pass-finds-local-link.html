<a href="resources/noop.html"></a>