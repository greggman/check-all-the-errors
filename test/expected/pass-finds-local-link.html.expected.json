{
  "numErrors": 0,
  "pages": {
    "http://localhost:8090/pass-finds-local-link.html": {
      "status": 200,
      "errors": []
    },
    "http://localhost:8090/resources/noop.html": {
      "status": 200,
      "errors": []
    }
  },
  "responses": [
    {
      "href": "http://localhost:8090/pass-finds-local-link.html",
      "resourceHref": "http://localhost:8090/pass-finds-local-link.html",
      "status": 200
    },
    {
      "href": "http://localhost:8090/resources/noop.html",
      "resourceHref": "http://localhost:8090/resources/noop.html",
      "status": 200
    }
  ]
}